.github/workflows/build-docker.yaml

```diff
@@ -0,0 +1,26 @@
+name: build-docker
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-go@v2
+      with:
+        go-version: 1.14.x
+    - uses: actions/checkout@v2
+    - name: Login to quay.io Docker Image Registry
+      uses: docker/login-action@v1
+      with:
+        registry: quay.io
+        username: "${{ secrets.QUAY_USER }}"
+        password: "${{ secrets.QUAY_PASS }}"
+    - run: make docker docker-publish
```

Comment:
Can we add something like https://github.com/thanos-io/thanos/blob/main/Makefile#L5 to build from main.